README.md

```diff
@@ -0,0 +1,126 @@
+## _InternalMidi_
+![](https://github.com/HaikuArchives/InternalMIDI/raw/master/documentation/gfx/icon.gif)
+![](https://github.com/HaikuArchives/InternalMIDI/raw/master/documentation/gfx/pecora.gif)
+
+![](https://github.com/HaikuArchives/InternalMIDI/raw/master/documentation/gfx/pixel.gif)
+
+# Introduction
+InternalMIDI creates a MIDI node for the internal General MIDI synthesizer of the BeOS.
+
+You can connect it to every BeOS MIDI program that supports the Midi2 Kit. With a nice panel you can change all the settings that the internal BeOS synthesizer accepts: Sound quality, different patches, pause, volume, reverb, and transposition.
+
+# License
+InternalMIDI is freeware. You may share InternalMIDI with as many people as you like, as long as the program is distributed with all parts, especially this documentation, intact. InternalMIDI may not be sold. You may freely distribute InternalMIDI together with your own software WITHOUT the authors permisson - although I would be glad to [hear from you](mailto:info@pecora.de)!  
```

Comment:
Remove the "- although I would be glad to [hear from you](mailto:info@pecora.de)!  "
Werner hasn't been active in Haiku since BeOS died.